{
	"id": "CVE-2019-6975",
	"vulnerabilities": [
		{
			"id": "CVE-2019-6975",
			"description": "Django 1.11.x before 1.11.19, 2.0.x before 2.0.11, and 2.1.x before 2.1.6 allows Uncontrolled Memory Consumption via a malicious attacker-supplied value to the django.utils.numberformat.format() function.",
			"severity": [
				{
					"type": "cvss_v30",
					"source": "security.alpinelinux.org",
					"cvss_v30": {
						"vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"base_score": 7.5,
						"base_severity": "HIGH",
						"temporal_score": 7.5,
						"temporal_severity": "HIGH",
						"environmental_score": 7.5,
						"environmental_severity": "HIGH"
					}
				}
			],
			"references": [
				{
					"source": "security.alpinelinux.org",
					"url": "https://security.alpinelinux.org/vuln/CVE-2019-6975"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "http://www.securityfocus.com/bid/106964"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://docs.djangoproject.com/en/dev/releases/security/"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://www.debian.org/security/2019/dsa-4476"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://www.djangoproject.com/weblog/2019/feb/11/security-releases/"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://www.openwall.com/lists/oss-security/2019/02/11/1"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://groups.google.com/forum/#%21topic/django-announce/WTwEAprR0IQ"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/66WMXHGBXD7GSM3PEXVCMCAGLMQYHZCU/"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/HVXDOVCXLD74SHR2BENGCE2OOYYYWJHZ/"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://seclists.org/bugtraq/2019/Jul/10"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://usn.ubuntu.com/3890-1/"
				}
			],
			"published": "2019-02-11T13:29:00Z",
			"modified": "2024-05-14T07:17:10.409802Z"
		}
	],
	"detection": [
		{
			"ecosystem": "alpine:3.10",
			"vulnerable": true,
			"package": {
				"name": "py-django"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "1.11.19-r0"
					}
				],
				"fixed": [
					"1.11.19-r0"
				]
			}
		},
		{
			"ecosystem": "alpine:3.11",
			"vulnerable": true,
			"package": {
				"name": "py3-django"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "1.11.19-r0"
					}
				],
				"fixed": [
					"1.11.19-r0"
				]
			}
		},
		{
			"ecosystem": "alpine:3.12",
			"vulnerable": true,
			"package": {
				"name": "py3-django"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "1.11.19-r0"
					}
				],
				"fixed": [
					"1.11.19-r0"
				]
			}
		},
		{
			"ecosystem": "alpine:3.9",
			"vulnerable": true,
			"package": {
				"name": "py-django"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "1.11.19-r0"
					}
				],
				"fixed": [
					"1.11.19-r0"
				]
			}
		}
	],
	"data_source": "alpine-osv"
}
